index.html

```diff
@@ -0,0 +1,160 @@
+            <a href="#"><img class="top-nav-item" src="assets/twitter .png" alt="twitter"></a>
+            <a href="#" class="top-nav-item">English</a>
+            <a href="#" class="top-nav-item">My Page</a>
+            <a href="#" class="top-nav-item">Logout</a>
+          </div>
+          <div class="bottom-nav-bar">
+            <a href="index.html"><img class="nav-bar-logo" src="assets/ailogo.png" alt="AI-logo"></a>
+            <div class="nav-bar-links">
+              <a class="bottom-nav-link" href="about.html">About</a>
+              <a class="bottom-nav-link" href="#partner">Partner</a>
+              <a class="bottom-nav-link" href="#">Join</a>
+              <a class="bottom-nav-link" href="#">Sponsor</a>
+              <a class="bottom-nav-link" id="ai" href="#">BUY TICKETS</a>
+            </div>
+          </div>
+          <!--  mobile-menu-->
+          <header class="mobile-head">
+            <nav class="mobile-nav">
+              <div class="mob-menu-links">
+                <img class="logo" src="assets/ailogo.png" alt="AI-logo">
+                <p class="mob-menu-para">Join <strong>India's </strong> vision to support <strong>United Nations</strong> in the field of artificial intelligence</p>
+                <a href="#" class="close-link">x</a>
+                <a class="bottom-nav-link" href="about.html"><strong>About</strong></a>
+                <a class="bottom-nav-link" href="#main"><strong>Program</strong></a>
+                <a class="bottom-nav-link" href="#speakers"><strong>Sponsor</strong></a>
+                <a class="bottom-nav-link enroll" href="#"><strong>Buy Ticket</strong></a>
+              </div>
+              <img class="nav-bar-icon" src="assets/hamburger .png" alt="hamburger-icon">
+            </nav>
+          </header>
+          <!-- finish mobile menu -->
+      </nav>
+      <div class="header-main">
+        <video class="header-video" autoplay muted loop>
+          <source src="assets/pexels-pavel-danilyuk-8328096 (2160p).mp4" type="video/mp4">
+        </video>
+        <div class="header-main-content">
+          <p class="header-main-salutation">"Hello! World Sharing"</p>
+          <h1 class="header-main-title">
+            <strong>AI For GOOD </br>INDIA SUMMIT</br>2023 </strong>
+          </h1>
+          <p class="header-main-description">
+            The Artificial Intellegence for Good India Summit is the leading action-oriented United Nations platform
+            promoting AI to advance health, climate, gender, inclusive prosperity, sustainable infrastructure, and other
+            global development priorities.
+          </p>
+          <h2 class="header-date">
+            <strong>24 JULY 2023, 17:00-18:30(CEST GENEVA)</strong>
+          </h2>
+          <p class="header-main-subdescription">
+            Join <strong>India's </strong> vision to support united nations in the field of artificial intelligence
+          </p>
+        </div>
+      </div>
```

Comment:
To make your page more semantic, Kindly take these welcome texts away from the header section😉. I suggest the header should only contain the nav and elements relating to navigations😉.  You can probably create an `intro` section with a section tag after the header and put these contents inside it👍.